extensions/cornerstone/src/services/CornerstoneCacheService/CornerstoneCacheService.ts

```diff
@@ -200,6 +200,8 @@ class CornerstoneCacheService {
     const volumeData = [];
 
     for (const displaySet of displaySets) {
+      const { hasVolumeData = false } = displaySet;
```

Comment:
I believe it should be `isOverlayVolume`, right? It's not about whether it has volume data or not, but rather about whether to overlay or not, I guess.

Reply:
Updated